src/modules/claims/claims.service.ts

```diff
@@ -683,11 +685,11 @@ export class ClaimsService {
 
     if (registrationTypes.includes(RegistrationTypes.OnChain)) {
       const { claimType: role, claimTypeVersion: version } = claim;
-      const expiry = expirationTimestamp || eternityTimestamp;
+      const expiry = Math.floor((expirationTimestamp || eternityTimestamp)/1000);
       const onChainProof = await this.createOnChainProof(
         role,
         version,
-        expiry,
+        Math.floor(expiry / 1000),
```

Comment:
Isn't this a second division by 1000? Isn't it being done already a couple lines above?